src/app/shell/details/achievements/achievements.component.html

```diff
@@ -1,7 +1,52 @@
+<div class="add-wrapper">
+  <p class="text">Тут ви можете додати досягнення.</p>
+  <a [routerLink]="['/create-achievement', workshop?.id]"><button class="btn" mat-raised-button>Додати досягнення</button></a>
+</div>
+
 <div class="card-wrapper" *ngIf="achievements?.length; else isEmptyList">
-  <app-achievement-card class="card" *ngFor="let achievement of achievements" [achievement]="achievement">
-  </app-achievement-card>
+  <mat-card class="position" *ngFor="let achievement of achievements">
+    <div class="card_achievement" fxLayout="column" fxLayoutAlign="start space-between">
+      <div class="wrap-for-details" fxLayout="row" fxLayoutAlign="start space-between">
+        <img class='image' src='../../../../assets/icons/award.svg'/>
+        <div>
+          <h3 class="achievement_title">{{achievementsTitle[achievement.achievementTypeId].name}}</h3>
+            <h4 class="card_title">Дата</h4> 
+            <p class="card_text">{{achievement.achievementDate | date: 'shortDate'}}</p>
+            <ng-container *ngFor="let child of achievement.children">
+              <h4 class="card_title" *ngIf="child">Учасники</h4> 
+              <p class="card_text">
+                {{child.firstName}} {{child.lastName}} {{child.middleName}}
+              </p>
+            </ng-container>            
+            <h4 class="card_title">Опис</h4> 
+
+            <mat-card-content>
+              <p class="achievement_descr" [ngClass]="{'show-more': showMore}">
+                {{(showMore) ? achievement.title : achievement.title | slice:0:100}}
+                <a *ngIf="achievement.title.length > 120" class='more_btn' 
+                  (click)="showMore = !showMore">...Більше</a>
+              </p>  
+            </mat-card-content>
+
+
+        </div> 
+        
+        <ng-container>
+          <div fxLayout="row" fxLayoutAlign="end" class="actions">
+            <div class="actions_btn" [routerLink]="['/create-achievement', achievement.id]">
+              <mat-icon class="achievement_icon">edit</mat-icon>
+            </div>
+            <div class="actions_btn" (click)="onDelete(achievement)">
+              <mat-icon class="achievement_icon">delete</mat-icon>
+            </div>
+          </div>
+        </ng-container>
+      </div>
+    </div>
+  </mat-card>
 </div>
 <ng-template #isEmptyList>
   <app-no-result-card [title]="noResultAchievements"></app-no-result-card>
 </ng-template>
+
+{{workshops | async | json}}
```

Comment:
Is there really a need to output the workshops as objects in raw form to the template?